Jenkinsfile

```diff
@@ -18,27 +22,86 @@ pipeline {
             // TODO CAL-296 refactor this stage from scripted syntax to declarative syntax to match the rest of the stages - https://issues.jenkins-ci.org/browse/JENKINS-41334
             steps{
                 parallel(
-                    linux: {
-                        node('linux-small') {
-                            timeout(time: 60, unit: 'MINUTES') {
+                        linux: {
```

Comment:
✏️  Not sure if we have any kind of indentation conventions for Jenkinsfiles, but the `parallel` "blocks" are indented by 8 instead of 4.